rust/gitxet/tests/integration_tests/test_xet_version_check.sh

```diff
@@ -2,19 +2,72 @@
 set -e
 set -x
 
+global_tmp_dir=/tmp
+
+if [[ ! -z ${TEMP} ]] ; then
+  global_tmp_dir="$TEMP"
+elif [[ ! -z ${TEMP_DIR} ]] ; then 
+  global_tmp_dir="$TMP"
```

Comment:
mismatch of what is checked and what is set.

Reply:
Fixed!